src/scripts/main.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = wall.querySelector('.spider');
+
+wall.addEventListener('click', (e) => {
```

Comment:
its cheat, look at conditions of the task. you should add evlistener to the `document`